tests/test_login.py

```diff
@@ -28,7 +28,7 @@ def setUp(self):
         self.username = "john.doe"
         self.password = "hunter2"
 
-        self.user = get_user_model().objects.create(username=self.username)
+        self.user = get_user_model().objects.create(username=self.username, is_staff=True)
```

Comment:
Why has this been changed?

Reply:
Only staff users have access to `"admin:index"`. For normal users there is no difference between a failed and successful login.

Reply:
Sounds good, thank you.